src/components/organisms/SectionContents/pricing.js

```diff
@@ -1,6 +1,12 @@
 //** Standard Frameworks */
 import * as React from 'react'
 
+//** Helpers */
+//** Personalization */
+import Text from "../../helper/Text";
+import { renderToString } from 'react-dom/server';
+import ReactHtmlParser from 'react-html-parser'; 
+
 /**
  * Section: Pricing
  */
```

Comment:
This is disgusting drug-induced crap, and should die
To put it with Linus words.